app/services/IdentityService.scala

```diff
@@ -201,9 +201,8 @@ object PersonalDataJsonSerialiser {
           "postcode" -> addr.postCode,
           "country" -> addr.country.fold(addr.countryName)(_.name)
         )
-      }),
```

Comment:
Do we need to update the IDAPI consent array instead?
I noticed we're still getting `receiveGnmMarketing` coming through in a POST form request and being set in SalesForce.

Reply:
good spot, since this subs frontend project is going away "soon" I would assume it's not a big issue.
Only paper+digital, and assisted sale would go through this route.  Paper+digital will be removed soon.  So if we miss out on consents via this route I don't think it would be a significant issue.